apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/api/PositiveRange.java

```diff
@@ -63,12 +65,28 @@ public void validate() {
     }
 
     @Override
-    public Number from() {
+    public Integer from() {
         return from;
     }
 
     @Override
-    public Number to() {
+    public Integer adjustBottomBoundary(Integer undefinedValue, Integer value) {
+        if (from.equals(undefinedValue)) {
+            from = value;
+        }
+        return from;
+    }
+
+    @Override
+    public Integer to() {
+        return to;
+    }
+
+    @Override
+    public Integer adjustTopBoundary(Integer undefinedValue, Integer value) {
+        if (to.equals(undefinedValue)) {
```

Comment:
undefined value could be internal value in PositiveRange object. PositiveRange could have a method isUndefined or smth like that

Reply:
incapsulated the undefined value;